src/main/java/duke/main/Parser.java

```diff
@@ -0,0 +1,103 @@
+package duke.main;
+
+import duke.command.*;
```

Comment:
I think we should avoid the wildcard import as much as possible, but I understand that Intellij does it automatically. It is best we manually and explicitly list out the classes we are importing.